### PR TITLE
Audio playback

### LINKED ISSUE #199: Fix Audio Playback and Align Audio Inline in AddNoteScreen Editor
Description:
In the AddNoteScreen, audio recording works, but playback doesn’t. Also, the audio player needs to be properly aligned in the editor.

Expected Behavior:
Audio Playback: The audio player should play recorded audio without issues.
Alignment and Cursor Behavior:
-  When a user records audio while typing, the audio should appear on a new line in the editor.
-   After the audio is inserted, the cursor should also start on a new line, ready for the user to keep typing.

Tasks:

- [x] Fix the audio playback issue so users can listen to recorded audio.
- [x] Make sure the audio player appears on a new line when there’s text before it.
- [x] Ensure the cursor starts on a new line after the audio.
- [x] Write test cases to confirm that playback and alignment work properly.

Acceptance Criteria:

- [x] Audio playback works smoothly.
- [x] Audio player appears on a new line if added while typing text.
- [x] Cursor starts on a new line after audio insertion.
- [x] All tests pass, and the app runs without issues.

### PR DESCRIPTION
**Pull request by AmarHadzic**
Fixes #199 

**What was changed**
The audio playback was fixed so that users can listen to recorded audio. Now it is an inline url that users can click on and open link, which leads the user to a chrome window and an audio player.

**Why was it changed**
Previously, users were not able to listen to recorded audio. Also, on the inline editor, nothing would show up after a user recorded audio. These issues led to poor user experience. Now users are able to playback their recorded audio, leading to an improved user experience.

**How was it changed**
Refactored and changed ```insertAudioToEditor``` function to handle and set the ```audioUri``` as an ```<a> </a>``` tag. This allows users to highlight and open the audio as a link. The audio link ```const audioLink = `${currentContent}<a href="${audioUri}">${audioUri}</a><br>`;``` captures previous content such as images, videos, or other audio and updates the inline editor. 

**Updated interface**
image

### CODE REVIEW

File: lib/navigation/AppNavigator.tsx [hunk 1]
```diff
@@ -12,12 +12,13 @@ import RegisterScreen from "../screens/loginScreens/RegisterScreen";
 import AddNoteScreen from "../screens/AddNoteScreen";
 import EditNote from "../components/EditNote";
 import OnboardingScreen from "../screens/OnboardingScreen";
+import VideoPlayerScreen from "../screens/VideoPlayer"; // Import the VideoPlayerScreen
```
Comment (ademDurakovic) [inline lib/navigation/AppNavigator.tsx:15]: Did you create this yourself why not get one from a library?

File: lib/screens/AudioPlayerScreen.tsx [hunk 2]
```diff
@@ -0,0 +1,31 @@
+import React from 'react';
```
Comment (ademDurakovic) [inline lib/screens/AudioPlayerScreen.tsx:1]: Is there not a library for this audiplayer?

File: styles/pages/NoteStyles.tsx [hunk 3]
```diff
@@ -2,10 +2,18 @@ import { Platform, StyleSheet } from "react-native";
...
+  .ProseMirror img {
+    max-width: 200px !important;
+    max-height: 200px !important;
+    object-fit: cover !important;
+    display: inline-block; /* Ensure images don't expand to fit container width */
+  }
+`;
 const NotePageStyles = () => {
   const { theme } = useTheme();
-
```
Comment (ademDurakovic) [inline styles/pages/NoteStyles.tsx:8]: Well done keeping the styling organized

File: lib/screens/AudioPlayerScreen.tsx [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+import { View, Text, Button } from 'react-native';
+import { Audio } from 'expo-av';
+
+const AudioPlayerScreen: React.FC<{ route: any, navigation: any }> = ({ route, navigation }) => {
```
Comment (ademDurakovic) [inline lib/screens/AudioPlayerScreen.tsx:5]: Seems like this and the video player are new components of this story? If working well done and good job separating it in a new file.

**Review by rcAsironman — CHANGES_REQUESTED** [review]
@AmarHadzic All checks have failed. Please review your work and resubmit it.

File: lib/screens/AudioPlayerScreen.tsx (hunk 2 above)
Comment (AmarHadzic) [reply lib/screens/AudioPlayerScreen.tsx:1]: Tentap does not support ones that are from libraries

File: lib/navigation/AppNavigator.tsx (hunk 1 above)
Comment (AmarHadzic) [reply lib/navigation/AppNavigator.tsx:15]: Yes, as tentap does not support inline

**Review by rcAsironman — CHANGES_REQUESTED** [review]
@AmarHadzic  could you please write test cases for the functionality you worked on?

**Comment by rcAsironman** [comment]
> @rcAsironman The test case was added. Also regarding the audio url link, you have to highlight it then scroll write on the pop up and click "open link". That is the only way it works with TenTap.

@AmarHadzic got it.

**Review by rcAsironman — APPROVED** [review]
lgtm;

**Review by yashb196 — APPROVED** [review]

**Pull request merged**